src/charm.py

```diff
@@ -32,6 +40,8 @@ def __init__(self, *args):
         super().__init__(*args)
         self._container_name = "login-ui"
         self._container = self.unit.get_container(self._container_name)
+        self._hydra_relation_name = "endpoint-info"
```

Comment:
We'll probably change this name to `hydra-endpoint-info` to be consistent, leaving a comment to keep track of it